services/clojars/clojars.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+const { BaseJsonService } = require('../base');
+const { NotFound } = require('../errors');
+const { version: versionColor} = require('../../lib/color-formatters');
```

Comment:
Being picky: inconsistent spacing around `versionColor` ^^

Reply:
Surprised we don't have an eslint rule for this.

Reply:
Good call. I was also surprised this isn't caught by a lint rule. Looks like we would need to configure ESLint's [object-curly-spacing](https://eslint.org/docs/rules/object-curly-spacing) rule. If we want to add a convention for this, there's going to be a bunch of other code knocking about that doesn't conform to it so lets do it in another PR.

Reply:
Actually, after taking a closer look into this, we do have a rule enabled in Prettier. Nevertheless, it's currently awaiting to be enabled as part of #1167.